### PR TITLE
Use rusty_v8 new crates.io release

### PR DESCRIPTION
**Pull request by leudz**
## 🌟 What is the purpose of this PR?

We had to use our own fork of rusty_v8 to have a specific function.
Upstream rusty_v8 merged this function so we don't need to use our fork.
And a new release is available with all we need.

## 🔗  Related links

- https://github.com/denoland/rusty_v8/issues/932

## 🔍 What does this change?

- Use the new `v8` crates.io release instead of our fork.

### 📜 Does this require a change to the docs?

No.

### CODE REVIEW

File: packages/engine/Cargo.toml [hunk 1]
```diff
@@ -46,9 +46,8 @@ tracing-appender = "0.2.0"
 tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.7", features = ["env-filter", "json"] }
 tracing-texray = { version = "0.1.2", optional = true }
-# Only temporary, use the crates.io release when ArrayBuffer::new_backing_store_from_ptr is available
-# https://github.com/denoland/rusty_v8/pull/926
-v8 = { git = "https://github.com/hashdeps/rusty_v8", rev = "0bf4d26961697591baf54bc4ceb0d15d3f195f86" }
+# Only temporary, use the next crates.io release
+v8 = { git = "https://github.com/denoland/rusty_v8" }
```
Comment (TimDiekmann) [inline packages/engine/Cargo.toml:50]: Does setting the `rev` field fixes the current issue?

File: packages/engine/Cargo.toml [hunk 2]
```diff
@@ -46,9 +46,8 @@ tracing-appender = "0.2.0"
 tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.7", features = ["env-filter", "json"] }
 tracing-texray = { version = "0.1.2", optional = true }
-# Only temporary, use the crates.io release when ArrayBuffer::new_backing_store_from_ptr is available
-# https://github.com/denoland/rusty_v8/pull/926
-v8 = { git = "https://github.com/hashdeps/rusty_v8", rev = "0bf4d26961697591baf54bc4ceb0d15d3f195f86" }
+# Only temporary, use the next crates.io release
```
Comment (TimDiekmann) [inline packages/engine/Cargo.toml:49]: Can we add the current version to the comment so it's easier to check, we can use the crates.io version?

**Review by TimDiekmann — CHANGES_REQUESTED** [review]

File: packages/engine/Cargo.toml (hunk 1 above)
Comment (leudz) [reply packages/engine/Cargo.toml:50]: I tried locally and it didn't fix the error.

Comment (TimDiekmann) [reply packages/engine/Cargo.toml:50]: That's sad ☹️

File: packages/engine/Cargo.toml (hunk 2 above)
Comment (leudz) [reply packages/engine/Cargo.toml]: Added the version https://github.com/hashintel/hash/pull/487/commits/de20f7a84176e91c8f8b108da4681a676095d289

**Comment by TimDiekmann** [comment]
I added the issue to the description.

---

> ### 🛡 What tests cover this?

> ### ❓ How to test this?

Could you update or delete these sections please?

**Comment by nonparibus** [comment]
@leudz @TimDiekmann Once this merges can we safely archive the https://github.com/hashdeps/rusty_v8 repo?

**Comment by TimDiekmann** [comment]
> @leudz @TimDiekmann Once this merges can we safely archive the https://github.com/hashdeps/rusty_v8 repo?

Yes! 🙂

**Review by TimDiekmann — APPROVED** [review]
Great!
You may want to close the issue: https://github.com/denoland/rusty_v8/issues/936

**Pull request merged**